py/client-ticking/build.gradle

```diff
@@ -44,59 +48,109 @@ def buildPyClientTicking = Docker.registerDockerTask(project, 'pyClientTicking')
         into layout.buildDirectory.dir('wheel')
     }
     dockerfile {
-        from('deephaven/cpp-client:local-build')
+        from('deephaven/cpp-client-py:local-build')
         runCommand("""mkdir -p \\
                         /out \\
-                        ${prefix}/log \\
-                        ${prefix}/src/py-client-ticking/src \\
-                        ${prefix}/src/py-client-ticking/in-wheels
+                        '${prefix}/log' \\
+                        '${prefix}/src/py-client-ticking/src' \\
+                        '${prefix}/src/py-client-ticking/in-wheels'
 
                    """)
         copyFile('setup.py', "${prefix}/src/py-client-ticking")
         copyFile('README.md', "${prefix}/src/py-client-ticking")
         copyFile('src/', "${prefix}/src/py-client-ticking/src/")
         copyFile('wheels/', "${prefix}/src/py-client-ticking/in-wheels")
-        runCommand("PREFIX=${prefix}; DEEPHAVEN_VERSION=${project.version};" +
+        runCommand("PREFIX='${prefix}'; WHEELS_SET='${wheelsSet}'; DEEPHAVEN_VERSION='${project.version}';" +
                    '''set -eux ; \
-                     cd "${PREFIX}/src/py-client-ticking"; \
-                     . "${PREFIX}/env.sh"; \
-                     MAKEFLAGS="-j${NCPUS}" \
-                       CFLAGS="-I${DHCPP}/include" \
-                       LDFLAGS="-L${DHCPP}/lib" \
-                       DEEPHAVEN_VERSION="${DEEPHAVEN_VERSION}" \
-                       python3 setup.py build_ext -i; \
-                     DEEPHAVEN_VERSION="${DEEPHAVEN_VERSION}" python3 setup.py bdist_wheel; \
-                     pip3 install in-wheels/*.whl; \
-                     pip3 install --force --no-deps dist/*.whl; \
-                     ln dist/*.whl /out; \
-                     cd /; \
-                     rm -fr "${PREFIX}/src/py-client-ticking"
+                      cd "${PREFIX}/src/py-client-ticking"; \
+                      . "${PREFIX}/env.sh"; \
+                      ORIG_PATH="$PATH"; \
+                      for spec in ${WHEELS_SET}; do \
+                        tag=`echo "$spec" | cut -d: -f 2`; \
```

Comment:
Seems a bit hacky - we could just pass through the data already structured into parts?